src/backend/tests/partaj/users/test_users.py

```diff
@@ -0,0 +1,63 @@
+from django.dispatch import receiver
+from django.test import RequestFactory, TestCase
+from django_cas_ng.models import SessionTicket
+from django_cas_ng.signals import cas_user_authenticated, cas_user_logout
+from partaj.core.factories import UnitFactory, UnitMembershipFactory, UserFactory
+from partaj.core.models.unit import UnitMembership, UnitMembershipRole
+from partaj.users.auth import CerbereCASBackend
+from partaj.users.signals import cas_user_authenticated_callback
+
+
+class UsersTestCase(TestCase):
+    """
+    User model related tests.
+    """
+
+    def test_signal_assign_new_user_to_unit_owner(self):
+        """
+        if unit membersip is empty the new user is assigned as member
+        """
+        unit = UnitFactory()
+        user = UserFactory(unit_name=unit.name)
+
+        cas_user_authenticated.send(
+            sender="manual",
+            user=user,
+            created=True,
+            username=user.username,
+        )
+
+        membership = UnitMembership.objects.get(user=user, unit=unit)
+        self.assertEqual(membership.role, UnitMembershipRole.OWNER, True)
```

Comment:
```suggestion
        self.assertEqual(membership.role, UnitMembershipRole.OWNER)
```

The third argument is just a message to display when the test case fails, almost never used in practice